src/pip/_internal/models/search_scope.py

```diff
@@ -77,11 +77,26 @@ def __init__(
     def get_formatted_locations(self):
         # type: () -> str
         lines = []
+        index_urls = []
```

Comment:
I think this is no longer needed? The content will always be identical to `self.index_urls`

Reply:
Hi @uranusjr 

The content is not the same as `self.index_urls`. For e.g, before it was 

```
lines.append(
                'Looking in indexes: {}'.format(', '.join(
                    redact_auth_from_url(url) for url in self.index_urls))
            )
```
So we call `redact_auth_from_url(url)` on `self.index_urls` and then add it to `lines`, which I mimic now with help of `index_urls`

Reply:
But the change is unnecessary in the current implementation. Now that I think about this again, the local variable is not the problem here, but you should use it in the warning as well.

Reply:
Okay I have renamed `index_urls ` to `redacted_index_urls`